src/jdk.jdeps/share/classes/com/sun/tools/jnativescan/ClassFileSource.java

```diff
@@ -75,14 +82,14 @@ public Stream<byte[]> classFiles(Runtime.Version version) throws IOException {
         }
     }
 
-    record ClassPathJar(Path path) implements ClassFileSource {
+    record ClassPathJar(Path path, Runtime.Version version) implements ClassFileSource {
         @Override
```

Comment:
The runtime version of was only being used by `ClassPathJar::classFiles`, so I've made it a property of `ClassPathjar` instead of a parameter to that method, which is not used by the other implementations.